src/scss/utils/elemental.scss

```diff
@@ -0,0 +1,11 @@
+.element-holder {
+  margin: 3rem 0;
+
+  &:first-child {
+    margin-top: 0;
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
```

Comment:
There might be a good reason why the margin on the last item is removed, but it means that when the screen width is collapsed down to mobile view the News section sits pretty tight up against the blocks. Could do with a little more space between blocks and News section.

Reply:
Done!